pennylane_lightning/lightning_qubit/lightning_qubit.py

```diff
@@ -226,9 +231,8 @@ def __init__(  # pylint: disable=too-many-arguments
 
             # Create the initial state. Internally, we store the
             # state as an array of dimension [2]*wires.
-            self._state = self._create_basis_state(0)
-            self._pre_rotated_state = self._state
-            self._c_dtype = c_dtype
+            self._state = _state_dtype(c_dtype)(self.num_wires)
+            self._pre_rotated_state = _state_dtype(c_dtype)(self.num_wires)
```

Comment:
Should we get rid of that?

Reply:
I removed it. We should consider removing it from other devices too, though.